department-of-veterans-affairs/va.gov-cms
# VACMS-2706: Updated Resources and Support Dashboard

…ding page block, updated resources and support page to include the new block in the header and updated url alias

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/2706

## Testing done


## Screenshots


## QA steps

- Login to preview environment with a user who has the ** Content creator - Resources and support ** role: 
  - Username: Laura.Walsh1@va.gov
  - Password: drupal8

- Navigate to the following page:
  - [ ] Confirm this page loads: /admin/content/resources-and-support

- Confirm page details:
  - [ ] Tab Label: Resources and support
  - [ ] 1st table title: Resources and support landing pages
  - [ ] 2nd table tile: Resources and support content

- Confirm table results details:
  - Resources and support landing pages:
    - [ ] Results are of content type landing page, single row, Title: Resources and support for your VA benefits and services
  - Resources and support content:
    - [ ] Results are of only R&S content types: Checklist, Step by step, Media list- Images, Media list - Videos, Q&A, FAQ, Learning center detail page.



## Definition of Done
